app/api/api_v1/routers/admin.py

```diff
@@ -317,6 +323,69 @@ def _start_ingest(
         )
 
 
+@r.post(
+    "/bulk-ingest/validate/cclw/law-policy",
+    response_model=ValidationResult,
+    status_code=status.HTTP_200_OK,
+)
+def validate_law_policy(
```

Comment:
New endpoint - uses the function `validate_law_policy_csv` which was factored out and now used in both the validate and ingest endpoints